### ISSUE TITLE
New XML formatter no longer allows formatting only selection.

### ISSUE THREAD

**Issue by garretwilson** [eclipse/wildwebdeveloper#1217]
One useful feature of the old formatter was the ability to select just a subset of the lines, and then `Ctrl+Shift+F` would only format the selected lines.

This is especially useful with bugs like #1216, where the formatter is arbitrarily breaking lines all over the place. But now I can't even localize the formatting to try to avoid this, because the formatter is ignoring the selected lines and formatting the entire file!

In summary the bugs are even breaking the workarounds to other bugs.

**Comment by angelozerr** [comment]
Is not the same issue than https://github.com/eclipse/wildwebdeveloper/issues/1205 ?

**Comment by vrubezhny** [comment]
Same here, it's the same as https://github.com/eclipse/wildwebdeveloper/issues/1205.

@garretwilson Please consider updating to M2E-Core 2.3.0 (https://download.eclipse.org/technology/m2e/snapshots/2.3.0/) and WWD 1.1.0 (https://download.eclipse.org/wildwebdeveloper/releases/1.1.0/) together in order to fix the issue.

Feel free to re-open in case of issue still persists after upgrading.
